src/Moryx.CommandCenter.Web/src/modules/container/Modules.tsx

```diff
@@ -104,21 +104,18 @@ class Modules extends React.Component<ModulesPropModel & ModulesDispatchPropMode
         this.state.MenuModel.MenuItems.forEach((menuItem) => {
             const module = this.props.Modules.filter(function (element: ServerModuleModel, index: number, array: ServerModuleModel[]): boolean { return element.name === menuItem.Name; })[0];
             routes.push(
-                <Route key={idx} path={menuItem.NavPath} exact={true}>
```

Comment:
could you just leave out the `exact` here?

Reply:
In version 6 of `react-router-dom` <Route exact> is gone and paths are relative :)